.github/workflows/windows.yml

```diff
@@ -81,9 +70,9 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
         with:
-          python-version: '3.11'
-          cache: 'pip'
-          cache-dependency-path: 'environment.yml'
+          python-version: "3.11"
+          cache: "pip"
+          cache-dependency-path: "environment.yml"
```

Comment:
Nit: it'd be nicer to do these unrelated formatting changes in one go in a separate PR and add a `git-rev-ignore` commit after it. I'm not a huge fan of formatting-only PRs, but it's much better than having them in every separate PR, so if you can't turn off your auto-formatting then I'd prefer that.